content/en/docs/contribute/_index.md

```diff
@@ -44,7 +44,7 @@ of roles and permissions.
 - [Suggest improvements](/docs/contribute/suggest-improvements/) to existing documentation and learn more about filing issues in GitHub.
 - Contribute a [small change](/docs/contribute/new-content/new-content/#small-changes) to existing content to learn the pull review process.
 - [Review pull requests](/docs/contribute/review/reviewing/) from other Kubernetes community members for accuracy and language.
-- Read the Kubernetes [content](/docs/contribute/style/content-guide/) and [style guides](/docs/contribute/style/style-guide/).
+- Read the Kubernetes [content](/docs/contribute/style/content-guide/) and [style guides](/docs/contribute/style/style-guide/) so you can leave informed comments.
```

Comment:
@celestehorgan , I read through this page. Found some small issues to fix:
Line 15, fix the format of the link.
Line 22, nit (you can contribute and not be a `member`), `Kubernetes documentation welcomes contributions from all members, both new and experienced!`
Possible change to something like:
The Kubernetes documentation project (?) welcomes contributors both new and experienced!

Line 69, reword end of sentence:
The meeting is always announced on the SIG Docs Slack channel and mailing list.